projects/frontend/cicd/install_data_pipelines.sh

```diff
@@ -33,7 +33,7 @@ else
 fi
 
 echo "Install Dependencies..."
-npm install
+npm install --omit=optional
```

Comment:
How did you verify it works ? Can you point out (in testing done section of the description) the relevant logs or whatever you used to verify the expected behaviour in. 


I am wondering because I see that in data pipelines/gui/package.json versatiledatakit/shared is not declared as optional so wouldn't still be installed from registry ...

Reply:
I tried adding shared as optional too, but it broke when I ran the install command. This tells me that the flag works, but I'll dig deeper into it. 🙂